tendenci/apps/directories/views.py

```diff
@@ -56,8 +56,41 @@ def details(request, slug=None, template_name="directories/view.html"):
 
 @is_enabled('directories')
 def search(request, template_name="directories/search.html"):
-    filters = get_query_filters(request.user, 'directories.view_directory')
+    user = request.user
+    username = request.user.username
+    filters = get_query_filters(user, 'directories.view_directory')
     directories = Directory.objects.filter(filters).distinct()
+
+    # Checking for view and change directory permission (staff)
+    can_view_directories = has_perm(user, 'directories.view_directory')
+    can_change_directories = has_perm(user, 'directories.change_directory')
+
+    can_view_pend = False
+    if can_view_directories and can_change_directories:
+        can_view_pend = True
```

Comment:
`can_view_pend = can_view_directories and can_change_directories`

Reply:
Changed in the newest commit.